specification/hugr.md

```diff
@@ -948,18 +995,21 @@ at runtime. In many cases this is desirable.
 
 To strike a balance then, every extension provides YAML that declares its opaque
```

Comment:
I don't think we mean just YAML here do we?

Reply:
Yes, the YAML seems less and less relevant!! Wilco (but keeping the "example" YAML)...

Reply:
Added "or equivalent Rust structs"